webpack/set-webpack-public-path-plugin/package.json

```diff
@@ -16,7 +16,7 @@
     "lodash": "~4.17.15",
     "uglify-js": "~3.0.28"
   },
-  "peerDependencies": {
+  "optionalPeerDependencies": {
```

Comment:
@iclanton Is this how to specify an optional peer dependency?  I thought it was designed [like in this doc](https://yarnpkg.com/configuration/manifest#peerDependenciesMeta) in order to be processed correctly by older package managers that don't understand the new feature.  The first real world example I've seen [was this](https://github.com/typescript-eslint/typescript-eslint/blob/adcee02d779f1faade877cb4a2692af5141ef09f/packages/eslint-plugin/package.json#L63) which works that way.